mllib/src/test/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluatorSuite.scala

```diff
@@ -40,4 +41,51 @@ class MulticlassClassificationEvaluatorSuite
   test("should support all NumericType labels and not support other types") {
     MLTestingUtils.checkNumericTypes(new MulticlassClassificationEvaluator, spark)
   }
+
+  test("Multiclass Classification Evaluator") {
+    val labelAndScores = spark.createDataFrame(Seq(
```

Comment:
Are these not tested in https://github.com/apache/spark/blob/master/mllib/src/test/scala/org/apache/spark/mllib/evaluation/MulticlassMetricsSuite.scala?